src/modules/trade/helpers/generate-trade.js

```diff
@@ -19,7 +19,9 @@ import store from 'src/store';
  * @param {Object} loginAccount
  * @param {Object} orderBooks Orders for market
  */
-export const generateTrade = memoize((market, outcome, outcomeTradeInProgress, loginAccount, orderBooks) => {
+export const generateTrade = memoize((market, outcome, outcomeTradeInProgress, orderBooks) => {
+  const { loginAccount } = store.getState();
```

Comment:
Is it intentional that `loginAccount` is no longer covered by the memoization?  Generally memoized functions shouldn't have internal calls to `getState`.